ansible/templates/zapd_block_height.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+dest_email={{ ALERT_EMAIL }}
+max_height_diff=2
+
+
+{% if DEPLOY_HOST == 'testnet.zap.me' %}
```

Comment:
just use the "remote_waves_node" here

Reply:
DEPLOY_HOST is a variable passed from the ansible_deploy.sh script. Do you want me to change the   ansible_deploy.sh script?

if [[ ( $DEPLOY_TYPE == "test" ) ]]
then
    DEPLOY_HOST=testnet.zap.me
    DEPLOY_USER=root
    REMOTE_WAVES_NODE=<name of the node>
    TESTNET=true
fi

Then pass the "REMOTE_WAVES_NODES" to the ansible-playbook?

Reply:
ok sure